apps/browser/qml/pages/components/PermissionListItem.qml

```diff
@@ -1,37 +0,0 @@
-import QtQuick 2.0
```

Comment:
Nice change that you're able to combine `PermissionListItem` with `BrowserListItem` here :1st_place_medal: